python/paddle/fluid/tests/unittests/white_list/op_check_grad_white_list.py

```diff
@@ -56,10 +56,6 @@
 
 # Special cases do not need to check grad
 NO_NEED_CHECK_GRAD_CASES = [
-    'TestLookupTableOpWithPadding',
-    'TestLookupTableOpWithTensorIdsAndPadding',
-    'TestLookupTableOpWithPadding',
-    'TestLookupTableOpWithTensorIdsAndPadding',
     'TestSeqMaxPool2DInference',
```

Comment:
Other cases in `NO_NEED_CHECK_GRAD_CASES` will use the `skip_check_grad_ci` wrapper later?

Reply:
Have fixed